setup.py

```diff
@@ -26,7 +26,7 @@
     package_data={"": ["*.txt", "*.rst"], "guillotina_s3storage": ["py.typed"]},
     install_requires=[
         'setuptools',
-        'guillotina>=5.0.0,<7',
+        'guillotina>=5.0.0,<6',
```

Comment:
I'm not sure about this pin here! Is this required for tests?

Reply:
@masipcat @bloodbare you're using this with 6 right?

Reply:
Yes!!!

Reply:
I'm not using s3 so I can't tell if it works with G6

Reply:
Yeps, the problem with Guillotina > 6 was a transitive dependency that was not correctly installed. The dependency was `async-asgi-testclient` and having the feeling that test requirements are kinda broken for Guillotina > 6 [1].

Ive decided to narrow down the version to the latest that most probably was tested, once I narrowed the version the transitive dependency problem got fixed.

I could add manually the dependency into the dependencies required by that `guillotina_s3_package`, but it would be a bit orthodox IMO since it is not a direct dependency.

I guess that the right solution would be fix first the Guillotina package, and later on, upgrade the versions that this package are compatible (if the PR gets accepted with the current proposal)


[1] https://github.com/plone/guillotina/blob/master/setup.py#L77

Reply:
when we moved g6 to asgi we stopped using the event_loop fixture from aiohttp and we started using pytest-asyncio that requires to mark async tests with decorator `@pytest.mark.asyncio`. I guess that's the problem with tests and G6?

Reply:
better to just cut a 5.x branch then instead of trying to make both cases work. We can revisit tests when we need g6 tests fixed...

Reply:
Sorry @vangheem , I'm a bit lost, what do you propose?

Reply:
I created a new 5.x branch and bumped master to 6.x.

Change this PR to be against the 5.x branch.

Reply:
Excelent!